installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_plugin_rules.json

```diff
@@ -11,5 +11,9 @@
     "aqua_aws_image_medium_vulnerabilities_scanned_rule",
 	"aqua_aws_image_high_vulnerabilities_scanned_rule",
 	"aqua_aws_image_critical_vulnerabilities_scanned_rule",
-	"aqua_aws_image_scanned_rule"
+	"aqua_aws_image_scanned_rule",
+	"contrast_critical_library_vulnerability_exists",
+	"contrast_high_library_vulnerability_exists",
+	"contrast_medium_library_vulnerability_exists",
+	"contrast_low_library_vulnerability_exists"
```

Comment:
and please make sure rule IDs here mathches rule IDs from SQL.
It won't break anything, right?

Reply:
no, It wont break anything, we are using policy UUID here